samples/scala/demo/conf/logback.xml

```diff
@@ -0,0 +1,18 @@
+<configuration scan="true" scanPeriod="5 seconds">
```

Comment:
This should be in a separate PR.

Reply:
I remove this file and PR in another branch.